Gemfile.lock

```diff
@@ -135,7 +135,7 @@ GEM
       sshkit (~> 1.2)
     capistrano-yarn (2.0.2)
       capistrano (~> 3.0)
-    capistrano3-puma (5.0.3)
+    capistrano3-puma (5.0.4)
```

Comment:
### [capistrano3-puma](https://github.com/seuros/capistrano-puma)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.0.3 → 5.0.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/seuros/capistrano-puma/compare/v5.0.3...v5.0.4).

These are the individual commits:
- (bf3548f) [fix: puma_systemctl_user default value (#319)](https://github.com/seuros/capistrano-puma/commit/bf3548f4f0b8d61749f2a73888817e57687bb846)
- (1868267) [release 5.0.4](https://github.com/seuros/capistrano-puma/commit/186826736ac83c550fd6129728284584b8c9c782)


</details>